src/test/kotlin/day15/repository_exam/RepositoryExam3Test.kt

```
@@ -0,0 +1,51 @@
+import org.junit.After
+import org.junit.Before
+
+import org.junit.Assert.*
+import org.junit.Test
+
+class RepositoryExam3Test {
+    private val test = TodoRepositoryImpl(MockTodoDatasourceImpl())
+
+    @Before
+    fun setUp() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)